### DISCUSSION TITLE
new install of homebrew w older installs

### DISCUSSION THREAD

**Discussion by LGitHub-sprout** [Homebrew/discussions#29] [Everyday usage] [answered]
hi,
hoping i understand the Homebrew install process enough to ask this question intelligently ;-p
i have older pkg installs in usr/local/bin as well as in other directories like (npm, git, gem, etc) in my home directory on a 2009 iMac running OS10.11.6.
i (believe i) understand that homebrew installs in /bin/bash and symlinks newly installed files to /usr/local.
questions is: would these symlinks be used automatically or would i have to change my $PATH (which i've never done before :/ )
many thanks!

**Comment by SMillerDev** [comment, marked as answer]
`/bin/bash` is a file. Homebrew installs everything to `/usr/local/Cellar` and symlinks it to the relevant unix directories in `/usr/local`. This is `bin`, `lib`, `share` etc. If you run `echo $PATH` and `/usr/local/bin` isn't in there you'll need to update it indeed.

**Reply by LGitHub-sprout** [reply]
ok, thank you for the clarifications.

**Comment by LGitHub-sprout** [comment]
got thank you for the clarifications <3
